helm/hocs-casework-search-indexer/values.yaml

```diff
@@ -10,10 +10,10 @@ app:
     requests:
       cpu: 2000m
       memory: 2048Mi
-  mode: MULTIPLE
-  newIndex: false
-  batchSize: 200
-  batchInterval: 500
+  mode: SINGULAR
```

Comment:
Does this need quotes too?

Reply:
I could add them for consistency but it doesn't need quoting because it's already a string

Reply:
I'm not overly fussed - just wanted to ask